app/cmd/plugin_upgrade.go

```diff
@@ -26,6 +28,9 @@ var pluginUpgradeOption PluginUpgradeOption
 func init() {
 	pluginCmd.AddCommand(pluginUpgradeCmd)
 	pluginUpgradeCmd.Flags().StringArrayVarP(&pluginUpgradeOption.Filter, "filter", "", []string{}, "Filter for the list, like: name=foo")
+	pluginUpgradeCmd.Flags().BoolVarP(&pluginUpgradeOption.All, "all", "", false, "upgrade all plugins")
```

Comment:
Please use `i18n.T('your description'` instead of literal. In addition, the description should be a complete sentence which meanings you need to let your sentence starts with a capital.